src/components/WebexInMeeting/WebexInMeeting.stories.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import jsonData from '../../data';
+import {WebexJSONAdapter} from '../../adapters';
+
+import {WebexInMeeting, WebexDataProvider} from '..';
+
+// Setup for the stories
+const stories = storiesOf('Webex InMeeting', module);
+const webexAdapter = new WebexJSONAdapter(jsonData);
+const wrapperStyle = {height: '500px', width: '800px', border: '1px solid black'};
+
+stories.add('remote and local media enabled', () => {
+  jsonData.meetings.remoteMedia.remoteAudio = {};
```

Comment:
Aren't these usually stored in the data itself?

Reply:
they are but coming from the other state to this one would not reset the properties to default. basically, it does not re-read the data.